R-packages/covidcast/R/plot.R

```diff
@@ -104,6 +107,80 @@ plot_choro = function(x, time_value = NULL, include = c(), range,
   geo = df$geo
   names(val) = geo
 
+  # Make background layer for MSA and HRR maps which are incomplete
+  if ((attributes(x)$geo_type == "msa") |
```

Comment:
```suggestion
  if ((attributes(x)$geo_type == "msa") ||
```

just a nitpick; `|` is vector elementwise OR, while `||` is logical OR with short-circuiting behavior